scripts/train_crnn.py

```
@@ -0,0 +1,334 @@
+        
+        train_loss /= train_batches
+        
+        # 検証
+        model.eval()
+        val_loss = 0.0
+        val_correct = 0
+        val_total = 0
+        val_batches = 0
+        
+        with torch.no_grad():
+            for images, labels, widths in tqdm(val_loader, desc=f"Epoch {epoch+1}/{epochs} - 検証"):
```

Ruff (0.12.2)

Loop control variable `widths` not used within loop body

Rename unused `widths` to `_widths`

(B007)